### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by jan-xyz** [grafana/grafonnet#213] [Q&A] [answered]
Now that [Grok is discontinued and archived](https://github.com/grafana/grok/) what does this mean for this project?

**Comment by Duologic** [comment, marked as answer]
This project is definitely not dead.

This OpenAPI/JSON schemas now come from [grafana-foundation-sdk](https://github.com/grafana/grafana-foundation-sdk). (Also mentioned in the README)

**Reply by jan-xyz** [reply]
Ahh! I missed that, grok was so deeply ingrained for me into this project. Sorry for jumping to quickly to conclusions without double-checking.
